secator/tasks/shodan.py

```
@@ -0,0 +1,163 @@
+		for name in (h.get('hostnames') or []) + (h.get('domains') or []):
+			if name and name not in seen:
+				seen.add(name)
+				yield Subdomain(host=name, domain=self._registered_domain(name), sources=['shodan'])
+		for key, label in (('org', 'shodan_org'), ('isp', 'shodan_isp'),
+						   ('asn', 'shodan_asn'), ('os', 'shodan_os')):
+			val = h.get(key)
+			if val:
+				yield Tag(name=label, value=str(val), match=ip_str, category='info', tags=['shodan'])
+		for cve in (h.get('vulns') or []):
+			yield Vulnerability(name=cve, id=cve, matched_at=ip_str, ip=ip_str,
+								provider='shodan', confidence='low', tags=['shodan'])
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)